e2e/suites/management/utils.go

```diff
@@ -121,6 +136,16 @@ func createSchedulerAndWaitForIt(
 
 		return len(svcAccs.Items) > 0
 	}, 5*time.Second, time.Second)
+
+	// creating secret used by the pods
+	kubeClient.CoreV1().Secrets(schedulerName).Create(context.Background(), &v1.Secret{
```

Comment:
If we don't create the secret in the namespace, the add rooms operations will fail because there will be errors while creating the instance on kubernetes

Reply:
> In this one which tool we will offer to the game team to create this kind of secret without touching in the scheduler namespace?

Currently, in this first release, the maestro does not provide means for creating those secrets. They must be created manually. We are adding it here for supporting maestro v9 schedulers, but we will need to better explore this feature (like supporting other tools like vault).